k8s/namespaces/em/em-hrs-api/aat.yaml

```diff
@@ -2,8 +2,11 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: em-hrs-api
+  annotations:
+    fluxcd.io/tag.java: glob:pr-375-*
 spec:
   values:
     java:
+      image: hmctspublic.azurecr.io/em/hrs-api:pr-375
```

Comment:
Should this be pr-375-7e65cda-20210915170059  ?

Reply:
flux will update each image  i believe to the latest PR